src/c++/perf_analyzer/concurrency_manager.cc

```diff
@@ -46,9 +46,9 @@ ConcurrencyManager::Create(
     const uint64_t sequence_id_range,
     const std::shared_ptr<ModelParser>& parser,
     const std::shared_ptr<cb::ClientBackendFactory>& factory,
-    std::unique_ptr<LoadManager>* manager)
+    std::shared_ptr<LoadManager>* manager)
```

Comment:
Why was this change made?

Reply:
testing a theory. I am getting seg faults in a test and I suspect it is because the ownership of the manager is moved around via a unique ptr

Reply:
What was the result of this theory?

Reply:
Wasn't the cause which makes sense in hindsight. The manager was owned in only one place and was safely destructed.